client/web/src/components/widgets/inputs/SwatchPairInput.vue

```diff
@@ -37,6 +38,21 @@
 			padding: 0;
 			box-sizing: border-box;
 			outline: none;
+			background: linear-gradient(45deg, #cccccc 25%, transparent 25%, transparent 75%, #cccccc 75%), linear-gradient(45deg, #cccccc 25%, transparent 25%, transparent 75%, #cccccc 75%),
+				linear-gradient(#ffffff, #ffffff);
+			background-size: 16px 16px;
+			background-position: 0 0, 8px 8px;
+
+			&::before {
+				content: "";
+				display: block;
+				width: 100%;
+				height: 100%;
+				background: var(--swatch-color);
+				background-size: 16px 16px;
+				background-position: 0 0, 8px 8px;
+				border-radius: 50%;
```

Comment:
To avoid any possible antialiasing problems, it is probably slightly more robust to get rid of this border-radius and add `overflow: hidden;` to the parent `button`.